src/masternode-sync.cpp

```diff
@@ -291,14 +291,14 @@ bool CMasternodeSync::SyncWithNode(CNode* pnode, bool isRegTestNet)
 {
     if (isRegTestNet) {
         if (RequestedMasternodeAttempt <= 2) {
-            pnode->PushMessage(NetMsgType::GETSPORKS); //get current network sporks
+            g_connman->PushMessageWithVersion(pnode, INIT_PROTO_VERSION, NetMsgType::GETSPORKS); //get current network sporks
```

Comment:
This message is sent after the handshake, shouldn't be using `INIT_PROTO_VERSION`.

Reply:
same as below, we can get rid of INIT_PROTO_VERSION doing https://github.com/PIVX-Project/PIVX/pull/1835#discussion_r483796406.

resolving comment and leaving only one open.